src/templates/en/2019/chapter.html

```diff
@@ -1,5 +1,7 @@
-{% extends "en/base.html" %}
+{{% extends "en/2019/base_chapter.html" %}}
```

Comment:
For consistency with other templates, use single braces as opposed to double braces, unless you're referencing a Jinja variable. (Here and throughout)
```suggestion
{% extends "en/2019/base_chapter.html" %}
```

Reply:
In this scenario, we unfortunately need the double braces. This isn't actually a jinja template, but instead used in the generation of the chapter jinja template. We do the following on line 56 of the `generate_chapters.py` script:

```python
file_to_write.write(template.format(body=body, metadata=metadata))
```

Because this is using the python string `format` command it will look for all braces to interpolate, which we don't want to have happen to the jinja code. The way one escapes those braces is the double-brace.

I agree that it's confusing, but I'm not sure what the solution is. We could move the code generation template out of the jinja templates directory and into a `generation/` directory? (although that feels clunky)

Reply:
Ah ok that makes sense. Thanks for explaining.